source/channels.yaml

```diff
@@ -410,6 +410,44 @@
               additionalProperties:
                 $ref: "#/definitions/ChannelMember"
 
+  '/teams/{team_id}/channels/{channel_id}/update_member_roles':
```

Comment:
I went for consistency with /teams/{team_id}/update_member_roles instead, but happy to change to be consistent with those other ones instead if you prefer?